src/theme/qml/ComboBox.qml

```diff
@@ -135,7 +167,7 @@ T.ComboBox {
         x: control.mirrored ? control.width - width : 0
         y: control.height
         width: control.width
-        height: contentItem.implicitHeight
+        height: contentItem.implicitHeight + theme.frameRadius * 8/3
```

Comment:
Again your favorite `theme.frameRadius`, it is also weird to multiply it by 8/3, which is 2,3333. Use a reasonable value/variable from theme.